### PR TITLE
Feature: Added Modal component

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added Modal component.

Live link: https://focused-euler-40cf42.netlify.app/

### CODE REVIEW

File: components/modal/styles.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    --primary-color: #4a4b4b;
+    --second-color: #9a16ce;
+    --third-color: #c65efa;
+    --white: rgb(231, 227, 227);
+}
+.modal-overlay {
+    position: absolute;
+    filter: blur(2.22px) grayscale(0.46);
+    height: 100vh;
+    width: 100vw;
```
Comment (viru2001) [inline components/modal/styles.css]: I think because of this horizontal scroll bar is coming, you can remove this

**Review by viru2001 — COMMENTED** [review]
you should try to remove the horizontal scroll bar . Rest everything looks good

File: components/modal/styles.css (hunk 1 above)
Comment (poojakelkar) [reply components/modal/styles.css]: thanks, viresh this one is resolved.

File: components/modal/styles.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
+:root {
+    --primary-color: #4a4b4b;
+    --second-color: #9a16ce;
+    --third-color: #c65efa;
+    --white: rgb(231, 227, 227);
+}
+.modal-overlay {
+    position: absolute;
```
Comment (shobit1337) [inline components/modal/styles.css]: modal should have position: fixed;

File: components/modal/styles.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    filter: blur(2.22px) grayscale(0.46);
+    height: 100%;
+    width: 100%;
```
Comment (shobit1337) [inline components/modal/styles.css]: insted of width:100% and height:100%,
You can use, inset: 0. 
It is a shorthand for top:0, bottom:0, right:0 and left:0

**Review by shobit1337 — APPROVED** [review]
Rest looks good to me!

File: components/modal/styles.css (hunk 2 above)
Comment (poojakelkar) [reply components/modal/styles.css]: Ok, Thanks for the suggestions.

File: components/modal/styles.css (hunk 3 above)
Comment (poojakelkar) [reply components/modal/styles.css]: Ok I will do that

**Pull request merged**